testsuite/python/coulomb_cloud_wall.py

```diff
@@ -108,7 +108,7 @@ def test_scafacos_p2nfft(self):
                 self.S.actors.add(Scafacos(bjerrum_length=1, method_name="p2nfft", method_params={
                                   "p2nfft_r_cut": 1.001, "tolerance_field": 1E-4}))
                 self.S.integrator.run(0)
-                self.compare("scafacos_p2nfft", energy=True)
+                self.compare("p3m_gpu", energy=False)
```

Comment:
The changed name only leads to a wrong log message.  However, I wonder why energy comparison was disabled.